CSVParser/README.md

```diff
@@ -0,0 +1,285 @@
+## CSV Parser Sample
+
+The CSV parser may be the simplest but practial parser in the world. In this sample, we will try to write one with the Lean programming language.
```

Comment:
That's a strong statement, how about "This is the simplest possible CSV parser you can write using Lean"